examples/aws-noaa-oisst-avhrr-only-example.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>can we write out words instead of <code>repr</code> I'm not sure that's going to be clear to people?</p><p> </p><blockquote>the full 40 years of data.</blockquote><p>", from 1981 to 2022."</p><p>Make it clear the printout doesn't actually say 40 anywhere, but that derived from reading the values.</p><p> </p><p><code>remote_protocol</code> isn't mentioned in the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>